Sources/StackDriverHandler.swift

```diff
@@ -0,0 +1,9 @@
+
+final class StackDriverHandler: StackDriverHandlerRepresentable {
```

Comment:
This class should implement `LogHandler` from `swift-log` library. you have to `import Logging`